File: src/main/java/com/yoyo/com/guess/GameFor.java
package com.yoyo.com.guess;

import java.util.Random;
import java.util.Scanner;

public class GameFor {
    public static void main(String[] args) {
        int random = new Random().nextInt(10)+1;
        Scanner scanner = new Scanner(System.in);
        for (int i =  5 ; i > 0 ; i--) {

            System.out.print("猜數字，剩下 "+i+" 次，請輸入數字: ");
            int num=scanner.nextInt();
            if (num > random) {
                System.out.println("低一點");
            } else if (num < random) {
                System.out.println("高一點");
            } else {
                System.out.println("恭喜猜中了!! 數字為: " + random);
                break;
            }

        }
    }
}

File: src/main/kotlin/com/yoyo/guess/GameFor.kt
package com.yoyo.guess

import java.util.*

fun main(args: Array<String>) {
    val random = Random().nextInt(10) + 1
    println("猜數字遊戲練習:")
    for (i in 1..5){
        print("剩餘$i 次，請輸入數字: ")
        val num = readLine()!!.toInt()
        if(num==-1){
            break
        }else if(num >random){
            println("低一點")
        }else if(num<random){
            println("高一些")
        }else{
            println("你猜對了!!! 該數字為$random")
            break
        }

    }
}